git.go

```diff
@@ -132,6 +133,23 @@ func CurrentBranch() (string, error) {
 	return strings.TrimSpace(string(result)), nil
 }
 
+// GetPullRequestURL creates a PR on Github and returns the URL
+// of the newly created PR. Returns an error
+// if you are not on a branch, or if you are not in a git repository.
+func GetPullRequestURL(title string) (string, error) {
+	cmd := exec.Command("hub", "pull-request", "-m", fmt.Sprintf("%s", title))
+	var out bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &out
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+	if err != nil {
+	    fmt.Println(fmt.Sprint(err) + ": " + stderr.String())
```

Comment:
Should use `os.Stderr` here.. for example `fmt.Fprintf(os.Stderr, "%s: %s", err.Error(), stderr.String())`